app/main.py

```diff
@@ -2,4 +2,53 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
I think better use `or` 
```suggestion
        self.coords = coords or [0, 0]
```